shell/app/modules/project/pages/plan-detail/batch-processing/index.tsx

```diff
@@ -63,12 +63,13 @@ const BatchProcessing = ({ afterDelete }: IProps) => {
         // eslint-disable-next-line no-case-declarations
         let queryParam = qs.stringify(omitBy({ selectProjectId, ...params }, isNull), { arrayFormat: 'none' });
         queryParam += qs.stringify(searchQuery, { arrayFormat: 'none' });
+        queryParam += `&${qs.stringify({ relationID: primaryKeys }, { arrayFormat: 'undefined' })}`;
```

Comment:
I can't find `arrayFormat: 'undefined'` option in https://www.npmjs.com/package/query-string
is it a legacy config?